pkg/exchange/coinbase/parse.go

```diff
@@ -0,0 +1,96 @@
+package coinbase
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/valyala/fastjson"
+)
+
+var parserPool fastjson.ParserPool
```

Comment:
Utilize the `"encoding/json"` package instead of fastjson due to performance issues.

Reply:
Done.